ci-operator/jobs/openstack-k8s-operators/nova-operator/openstack-k8s-operators-nova-operator-master-presubmits.yaml

```diff
@@ -203,6 +203,68 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )nova-operator-build-deploy,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - ^master$
+    - ^master-
+    cluster: build05
+    context: ci/prow/nova-operator-build-deploy-tempest
+    decorate: true
+    labels:
+      ci.openshift.io/generator: prowgen
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openstack-k8s-operators-nova-operator-master-nova-operator-build-deploy-tempest
+    optional: true
```

Comment:
given this is optional im ok with approving for now

some of the test coverage is questionable since the tempest test expect at least 1 compute but are not checking that

so some things are passing like 

tempest.api.compute.admin.test_hosts_negative.HostsAdminNegativeTestJSON.test_show_host_detail_with_non_admin_user

this deployment does not have a compute node so there is no host in the response  and this is passing 

so some of the coverage is currently providing false coverage.